backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5684730_gh15100_Handle_Multiple_SalesRep.sql

```diff
@@ -0,0 +1,15 @@
+-- 2021-02-15T12:54:07.014Z
+-- #298 changing anz. stellen
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Reference_Value_ID,AD_Table_ID,ColumnName,ColumnSQL,Created,CreatedBy,DDL_NoForeignKey,Description,EntityType,FacetFilterSeqNo,FieldLength,Help,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutoApplyValidationRule,IsAutocomplete,IsCalculated,IsDimension,IsDLMPartitionBoundary,IsEncrypted,IsFacetFilter,IsForceIncludeInGeneratedModel,IsGenericZoomKeyColumn,IsGenericZoomOrigin,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsRangeFilter,IsSelectionColumn,IsShowFilterIncrementButtons,IsStaleable,IsSyncDatabase,IsTranslated,IsUpdateable,IsUseDocSequence,MaxFacetsToFetch,Name,SelectionColumnSeqNo,SeqNo,Updated,UpdatedBy,Version) VALUES (0,572811,1063,0,18,190,540270,'SalesRep_ID','(select bp.SalesRep_ID from C_BPartner bp where bp.c_bpartner_id = C_Invoice_Candidate.bill_bpartner_id)',TO_TIMESTAMP('2021-02-15 14:54:06','YYYY-MM-DD HH24:MI:SS'),100,'N','','de.metas.invoicecandidate',0,10,'','Y','N','Y','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N',0,'Kundenbetreuer',0,0,TO_TIMESTAMP('2021-02-15 14:54:06','YYYY-MM-DD HH24:MI:SS'),100,0)
```

Comment:
this is not valid, as the `salesrep_id` it's coming from the `c_order` when we are creating the invoice => that should be our source for the aggregation

Reply:
correction done